packages/fxa-shared/subscriptions/types.ts

```diff
@@ -165,9 +165,12 @@ export type WebSubscription = Pick<
       'incomplete' | 'incomplete_expired'
     >;
     subscription_id: Stripe.Subscription['id'];
+    promotion_amount_off?: string | null;
```

Comment:
`string` or `number`? We have this somewhere above:
```js
  promotion_amount_off: isA
    .number()
    .min(0)
```

Reply:
Womp, copy pasta backfired. Thanks Peter!